frontend/src/concepts/modelRegistry/utils.ts

```diff
@@ -0,0 +1,36 @@
+export type ObjectStorageFields = {
+  endpoint: string;
+  bucket: string;
+  region?: string;
+  path: string;
+};
+
+export const objectStorageFieldsToUri = (fields: ObjectStorageFields): string => {
+  const { endpoint, bucket, region, path } = fields;
+  const searchParams = new URLSearchParams();
+  searchParams.set('endpoint', endpoint);
```

Comment:
Why don't we test first if the attributes are correct to sanetize it? I know we get the fields as mandatory but what happens if somehow endpoint can be input as an empty string? the uri is malformed or it's not set as a search param?